qiita_ware/ebi.py

```diff
@@ -993,6 +744,8 @@ def generate_demultiplexed_fastq(self, rewrite_fastq=False):
                     if wrote_sequences:
                         demux_samples.add(s)
                     else:
+                        del(self.samples[s])
```

Comment:
Will add a note but they are required as if we do not delete the sample from the sample/prep template the methods will try to submit them even if they do not have a fastq.gz file AKA the no sequences error.